api/handlers/space.go

```diff
@@ -204,6 +206,14 @@ func (h *Space) getIsolationSegment(r *http.Request) (*routing.Response, error)
 	return routing.NewResponse(http.StatusOK).WithBody(struct{}{}), nil
 }
 
+func (h *Space) getRunningSecurityGroups(r *http.Request) (*routing.Response, error) {
+	return routing.NewResponse(http.StatusOK).WithBody(presenter.ForList(presenter.ForSecurityGroup, repositories.ListResult[repositories.SecurityGroupRecord]{}, h.apiBaseURL, *r.URL)), nil
+}
+
+func (h *Space) getStagingSecurityGroups(r *http.Request) (*routing.Response, error) {
+	return routing.NewResponse(http.StatusOK).WithBody(presenter.ForList(presenter.ForSecurityGroup, repositories.ListResult[repositories.SecurityGroupRecord]{}, h.apiBaseURL, *r.URL)), nil
+}
+
```

Comment:
@danail-branekov should I keep it like that in here or should I use the empty present because security groups is not yet really supported?